### PR TITLE
Cbrd 23662 json arrayagg

### PR DESCRIPTION
**Pull request by hgryoo**
Dtdyfyfhfu

### CODE REVIEW

File: ko/sql/function/analysis_fn.rst [hunk 1]
```diff
@@ -2048,3 +2048,65 @@ VAR_SAMP
...
                 3  'Wane'                   9.900000000000000e+01     5.435000000000000e+02
+
+.. _fn-json-arrayagg:
+
+JSON_ARRAYAGG
+===================================
+
+.. function:: JSON_ARRAYAGG (json_val)
+
+  평가된 행으로부터 JSON 배열을 만드는 집계 함수이다.
```
Comment (charischau) [inline ko/sql/function/analysis_fn.rst:2059]: this line looks weird.

File: ko/sql/function/analysis_fn.rst [hunk 2]
```diff
@@ -2048,3 +2048,65 @@ VAR_SAMP
...
+
+.. code-block:: sql
+
+    CREATE TABLE t_score(name VARCHAR(10), score INT);
+    INSERT INTO t_score VALUES
```
Comment (charischau) [inline ko/sql/function/analysis_fn.rst:2064]: review1

**Review by charischau — CHANGES_REQUESTED** [review]

File: ko/sql/function/analysis_fn.rst (hunk 1 above)
Comment (hgryoo) [reply ko/sql/function/analysis_fn.rst:2059]: I do t s

File: ko/sql/function/analysis_fn.rst (hunk 2 above)
Comment (charischau) [inline ko/sql/function/analysis_fn.rst:2064]:
```suggestion
    CREATE TABLE t_score(name VARCHAR(10), score INT);
    INSERT INTO t_score VALUE
```

**Review by charischau — APPROVED** [review]

File: ko/sql/function/analysis_fn.rst [hunk 3]
```diff
@@ -241,7 +241,7 @@ AVG
 
 .. code-block:: sql
 
-    SELECT host_year, nation_code, gold,
+  
         AVG(gold) OVER (PARTITION BY nation_code ORDER BY host_year) avg_gold
     FROM participant WHERE nation_code like 'AU%';
```
Comment (charischau) [inline ko/sql/function/analysis_fn.rst:246]: fjf

File: ko/sql/function/analysis_fn.rst [hunk 4]
```diff
@@ -2048,3 +2048,65 @@ VAR_SAMP
                 3  'Lee'                    9.300000000000000e+01     5.435000000000000e+02
                 3  'Sara'                   4.300000000000000e+01     5.435000000000000e+02
                 3  'Wane'                   9.900000000000000e+01     5.435000000000000e+02
+
+.. _fn-json-arrayagg:
+
+JSON_ARRAYAGG
```
Comment (charischau) [inline ko/sql/function/analysis_fn.rst:2054]: hfhk

**Review by charischau — COMMENTED** [review]
summary

**Pull request closed without merging**